lib/notify.js

```diff
@@ -1,15 +1,23 @@
 const notifier = require("node-notifier");
 const path = require("path");
 const config = require("./config");
+const { logger } = require("./logger");
 
 module.exports = (message) => {
   if (config["TD_NOTIFY"]) {
-    return notifier.notify({
-      title: "TestDriver.ai",
-      message,
-      timeout: 5,
-      icon: path.join(__dirname, "images", "icon.png"),
-      sound: false,
-    });
+    logger.debug(`${__filename}: (${JSON.stringify(message)})`);
```

Comment:
I love the filename output. I think we have a similar log already? 

https://github.com/testdriverai/testdriverai/blob/14a934d118cc864c1fc80239b6384f8ef4558b0a/lib/commander.js#L34